build/config.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# 
+# LiskHQ/lisk-build
+# Copyright (C) 2017 Lisk Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+
+# shellcheck disable=SC2034
+
+VERSION="$VERSION"
+OS=$(uname)
+ARCH="${ARCH:-$( uname -m )}"
+BUILD_NAME="lisk-$VERSION-$OS-$ARCH"
+TARGET=""
+
+ncpu=$( grep -c processor /proc/cpuinfo )
+ncpu=$(( ncpu + 0 ))
+tcpu=$(( ncpu / 2 + 1 ))
+[[ $tcpu -gt 8 ]] && tcpu=8
+MAKEOPTS="-j${tcpu}"
+
+LISK_DIR="$VERSION"
+LISK_FILE="lisk-$VERSION.tgz"
+LISK_NETWORK="$LISK_NETWORK"
+LISK_URL="https://downloads.lisk.io/lisk/$LISK_NETWORK/$VERSION/$LISK_FILE"
+
+LISK_SCRIPTS_VERSION="0.4.0"
+LISK_SCRIPTS_SHA256SUM="03ddb4d651bba12906c77a6f1c3a6931c0e227bcd20cf44d409c0cd59c8d7b12"
+LISK_SCRIPTS_DIR="lisk-scripts-$LISK_SCRIPTS_VERSION"
+LISK_SCRIPTS_FILE="$LISK_SCRIPTS_DIR.tar.gz"
+LISK_SCRIPTS_URL="https://github.com/LiskHQ/lisk-scripts/archive/v$LISK_SCRIPTS_VERSION.tar.gz"
+
+NODE_VERSION="8.12.0"
```

Comment:
Is it possible to extract this version from `package.json` key `engines.node`?

Reply:
I was thinking about taking the version in `.nvmrc` actually: https://github.com/LiskHQ/lisk-build/issues/351